bin/create.js

```diff
@@ -37,6 +50,10 @@ var tokenOptions = {
         'default': 'default',
         type: 'string',
     },
+    auth0: {
+        description: 'Authenticate webtask with Auth0: --auth0 <clientID> <client-secret> <auth0-domain>',
```

Comment:
This should show quotes around <clientID> <client-secret> <auth0-domain>. I wonder if a better pattern would be to use commas or separate options for each of these?